README.md

```diff
@@ -210,40 +205,42 @@ Follow the howto article [HOW-to guides: Manage Models in the repository](https:
 
 Based on the existing samples, create a new device to implement the [Themorstat Model](./models/Thermostat.json).
 
-The device should imlement the next interfaces:
+The device should implement the next interfaces:
+
+- Handle the `reboot` command and send the `workingset` as telemetry in the `Diagnostics` interface
```

Comment:
Nit on diagnistics DTDL.  Please remove the `"commandType": "synchronous"`.   We're about to make asychronous go away as a supported concept from DTDL for PnP.  We'll default to sychronous as always and not having `commandType` surface in DTDL sample makes it less likely someone will go poking around in it.